src/randstrobes.cpp

```diff
@@ -154,6 +159,15 @@ Randstrobe RandstrobeIterator::get(unsigned int strobe1_index) const {
     uint64_t min_val = std::numeric_limits<uint64_t>::max();
     Syncmer strobe2 = strobe1;
 
+    if (syncmers.size() < w_start) {
+        return Randstrobe{
+            randstrobe_hash(strobe1.hash, 0, aux_len),
```

Comment:
Would it be ok to use `strobe1.hash` as the second parameter?
```suggestion
            randstrobe_hash(strobe1.hash, strobe1.hash, aux_len),
```
We already generate randstrobes that represent a syncmer paired with itself when the second syncmer is too far downstream; this would make both cases consistent with each other.

(Actually, the code would become much simpler also because the entire `if (syncmers.size() < w_start)` block could be deleted and we would rely on the for loop being executed zero times, just like when the second syncmer is further away than `max_position`.)

Reply:
I’ve now made that change in commit 9c3768364b3f3e9ef981516b2c4fdac2d8a5bf53. It was necessary to avoid some spurious NAMs (which made `unmappable.41` mappable). See the commit message for an explanation.

Reply:
Oh I see! Passing a 0 was a bug that I introduced! Good catch